dist/tcti-tabrmd.pc.in

```diff
@@ -1,7 +1,7 @@
 Name: tcti-tabrmd
 Description: TCTI library for communicating with the TPM2 access broker / resource manager daemon (tabrmd).
-URL: https://github.com/01org/??
+URL: https://github.com/01org/tpm2-abrmd
 Version: @VERSION@
-Requires: sapi
+Requires: sapi glib-2.0 gio-2.0
```

Comment:
To make this concrete: it's not about the packaging, it's about `pkg-config`

```
$ pkg-config --cflags glib-2.0
-I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include
$ pkg-config --libs glib-2.0
-lglib-2.0
```
vs
```
$ pkg-config --cflags gio-2.0
-pthread -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include
$ pkg-config --libs gio-2.0
-lgio-2.0 -lgobject-2.0 -lglib-2.0 
```